### PR TITLE
Refresh debian/ packaging

### PR DESCRIPTION
**Pull request by OddBloke**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -10,25 +10,23 @@ Build-Depends: debhelper (>= 9),
...
                python3-all-dev,
                python3-flake8,
                python3-jsonschema,
                python3-nose,
                python3-pyudev,
                python3-setuptools,
+               python3-testtools,
                python3-yaml
-Standards-Version: 3.9.5
+Standards-Version: 4.3.0
```
Comment (panlinux) [inline debian/control]: Could you clarify if no changes were needed, or if some were, which ones? In d/changelog is enough.

Comment (OddBloke) [reply debian/control]: The only change required was the priority (although I changed that before running through https://www.debian.org/doc/packaging-manuals/upgrading-checklist.txt due to a lintian warning). I've moved that entry to be under the Standards-Version entry; does that make it clear enough?

**Comment by mwhudson** [comment]
This all looks fine, thanks.

File: debian/control (hunk 1 above)
Comment (panlinux) [reply debian/control]: sure

File: debian/control [hunk 2]
```diff
@@ -10,25 +10,23 @@ Build-Depends: debhelper (>= 9),
...
 Homepage: https://github.com/CanonicalLtd/probert
-X-Python3-Version: >= 3.3
 Vcs-Browser: https://github.com/CanonicalLtd/probert
 Vcs-Git: https://github.com/CanonicalLtd/probert.git
 
 Package: probert
 Architecture: any
-Depends: python3-all,
-         python3-pyudev,
+Depends: python3-pyudev,
```
Comment (raharper) [inline debian/control]: python3-jsonschema
libnl-genl-3-200
libnl-3-200

**Pull request merged**